### PR TITLE
mrc-1914 - fix import button loading endlessly

### PR DESCRIPTION
**Pull request by carbowizzy**
This PR fixes error which was generated when a user clicks "Import" to import files from ADR when no file is selected, it hangs at the "Importing files - this may take several minutes screen.

### CODE REVIEW

File: src/app/static/src/app/components/adr/SelectDataset.vue [hunk 1]
```diff
@@ -210,6 +210,7 @@
                     outOfDate: false} : null
             },
             async importDataset() {
+                if(this.newDatasetId){
```
Comment (hillalex) [inline src/app/static/src/app/components/adr/SelectDataset.vue:213]: This needs auto-formatting/proper indenting! And a unit test. Can we also mark the button as disabled when `this.newDatasetId` is null, so the user can't even click it (and add a unit test for this too)?

**Review by hillalex — CHANGES_REQUESTED** [review]

File: src/app/static/src/tests/components/adr/selectDataset.test.ts [hunk 2]
```diff
@@ -588,7 +588,7 @@ describe("select dataset", () => {
 
         await Vue.nextTick();
 
-        expect(rendered.findAll(LoadingSpinner).length).toBe(1);
+        expect(rendered.findAll(LoadingSpinner).length).toBe(0);
```
Comment (hillalex) [inline src/app/static/src/tests/components/adr/selectDataset.test.ts:591]: why has this been changed? it changes the meaning of this test...

**Review by hillalex — CHANGES_REQUESTED** [review]

File: src/app/static/src/app/components/adr/SelectDataset.vue (hunk 1 above)
Comment (carbowizzy) [reply src/app/static/src/app/components/adr/SelectDataset.vue:215]: I auto-format, I think the issue has to do with how IntelliJ and VSCode do formatting. I will explore IntelliJ for formatting.

File: src/.idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -19,18 +19,9 @@
       </option>
       <option name="LBRACE_ON_NEXT_LINE" value="true" />
     </JetCodeStyleSettings>
-    <VueCodeStyleSettings>
```
Comment (hillalex) [inline src/.idea/codeStyles/Project.xml:22]: This should not be removed - it contains a rules for indenting Vue files...

**Review by hillalex — COMMENTED** [review]
The build is failing - I know it's easy to miss sometimes but as part of our PR protocol you should only request a review once you've confirmed that the build is passing!

**Review by hillalex — APPROVED** [review]

**Pull request merged**